lua/core/autopairs.lua

```diff
@@ -1,51 +1,67 @@
 -- if not package.loaded['nvim-autopairs'] then
 --   return
 -- end
-local status_ok, autopairs = pcall(require, "nvim-autopairs")
-if not status_ok then
-  return
+local M = {}
+
+M.config = function()
+  O.plugin.autopairs = {
+    config = false,
+  }
 end
-local npairs = require "nvim-autopairs"
-local Rule = require "nvim-autopairs.rule"
 
--- skip it, if you use another global object
-_G.MUtils = {}
+M.setup = function()
+  if O.plugin.autopairs.config then
+    return O.plugin.autopairs.config
+  end
+
+  local status_ok, autopairs = pcall(require, "nvim-autopairs")
+  if not status_ok then
+    return
+  end
+  local npairs = require "nvim-autopairs"
+  local Rule = require "nvim-autopairs.rule"
```

Comment:
This is from the old code, but aren't `npairs` and `autopairs` basically the same variable?

Reply:
It looks like a mistake but Chris added it last week.  It's possible this is related to hot reloading of the config.  I'll have to ask about it.  But if it needs to change, that will be a separate pr.